package.json

```diff
@@ -86,7 +86,7 @@
     "broccoli-string-replace": "^0.1.1",
     "chalk": "^1.1.0",
     "electron-forge": "^2.8.0",
-    "electron-protocol-serve": "^1.1.0",
+    "electron-protocol-serve": "^1.2.0-2",
```

Comment:
This should work, but I think this was [republished as 1.3.0](https://github.com/pichfl/electron-protocol-serve/releases) during troubleshooting, so we may as well bump this to match.